src/elements/expandable-list/expandable-list.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect, useRef } from 'react'
+
+import styles from './expandable-list.css'
+
+import { Button } from 'elements'
+import { classNames } from 'utils'
+
+const ExpandableListItem = ({ children, ...htmlProps }) => (
+  <li {...htmlProps}>{children}</li>
+)
+
+const ExpandableList = ({ children, className, ...htmlProps }) => {
```

Comment:
I think it's too early to generalise the name because the component may become used inappropriately and difficult to understand what it is. Currently, this component is only the author list, it expands in a very specific way and displays in a very specific way too.

I find [these guidelines](https://namingthings.donedone.com) very useful to pick up good naming practices.

Reply:
I imagined generalising component even more once we need it. For example support, different separators, different `show more` possitions etc. Currently, it is just authors list as you said.